multirowgroup.go

```diff
@@ -0,0 +1,246 @@
+package parquet
+
+import (
+	"io"
+)
+
+// MultiRowGroup wraps multiple row groups to appear as if it was a single
+// RowGroup. RowGroups must have the same schema or it will error.
+func MultiRowGroup(rowGroups []RowGroup) (RowGroup, error) {
+	if len(rowGroups) == 0 {
+		return nil, nil
```

Comment:
This should return a non-nil value, otherwise we are creating fertile ground for application to trigger panics dereferencing nil pointers:
```suggestion
		return emptyRowGroup{}, nil
```